<launch>
	<node pkg ="home_monitoring_pkg" type="CameraNode.py" name="cameraNode" output="screen"/>
	<node pkg ="home_monitoring_pkg" type="PredictionNode.py" name="PredictionNode" output="screen"/>
	<node pkg ="home_monitoring_pkg" type="OverlayNode.py" name="OverlayNode" output="screen"/>
	<node pkg ="home_monitoring_pkg" type="DisplayNode.py" name="DisplayNode" output="screen"/>
	<node pkg ="home_monitoring_pkg" type="TelegramBot.py" name="TelegramBot" output="screen"/>
</launch>
